adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -909,12 +911,21 @@ def _wait_for_msg(self, timeout=0.1):
                     "Unexpected PINGRESP returned from broker: {}.".format(sz)
                 )
             return MQTT_PINGRESP
-        if res[0] & 0xF0 != 0x30:
+
+        if res[0] & 0xF0 != MQTT_PUBLISH:
```

Comment:
Since it was made a `const()`, could you check if we use `0x30` elsewhere in the file and replace with `MQTT_PUBLISH` ?

Reply:
Sure, done.